### ISSUE TITLE
Preserve image order from source data provider

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/ala-bie-hub#84]
As user of the ALA and other data source platforms such as iNaturalist, I want the ALA to preserve the order I set for images in the source platform.

**Background**
Currently records in the ALA with images coming from data sources such as iNatdon't preserve the order the images had from source.

See example below

![Screenshot 2023-08-25 at 2 52 37 pm](https://github.com/AtlasOfLivingAustralia/ala-bie-hub/assets/8326394/5846e36b-4aa0-4b02-b698-8d1e6b2ffe7e)
3rd image from iNaturalist  displaying in BIE for _Hibiscus diversifolius_
https://bie.ala.org.au/species/https://id.biodiversity.org.au/node/apni/2896715#gallery

![Screenshot 2023-08-25 at 2 52 26 pm](https://github.com/AtlasOfLivingAustralia/ala-bie-hub/assets/8326394/3068c47f-dd10-4d16-9c22-0291a156c4f0)
iNaturalist images for observation 175917918 (_Hibiscus diversifolius_)
https://www.inaturalist.org/observations/175917918 

A more detailed analysis is in https://github.com/AtlasOfLivingAustralia/bie-plugin/issues/247

**Requirements**
Images should preserve the order from the source platform.

While the most common scenario is for images coming from iNaturalist it is important to understand if this affects/apply to other sources: FrogID, NatureMapr, eBird, etc.

**Comment by peggynewman** [comment]
@sadeghim could you pull out the source data from the DwCA for this record so we can see whether that image order data is provided to us in the first place?

**Comment by sadeghim** [comment]
@peggynewman Just a note that I don't think DwCA has an order field or assumptions for extensions or even core records.  
And yes those 3 images are in order in the media.csv:
```
175917918,StillImage,image/jpeg,https://inaturalist-open-data.s3.amazonaws.com/photos/305781052/original.jpeg,https://www.inaturalist.org/photos/305781052,2023-07-30T12:42:05-07:00,Thomas Mesaglio,iNaturalist,http://creativecommons.org/licenses/by/4.0/,Thomas Mesaglio,305781052
175917918,StillImage,image/jpeg,https://inaturalist-open-data.s3.amazonaws.com/photos/305781029/original.jpeg,https://www.inaturalist.org/photos/305781029,2023-07-30T12:42:02-07:00,Thomas Mesaglio,iNaturalist,http://creativecommons.org/licenses/by/4.0/,Thomas Mesaglio,305781029
175917918,StillImage,image/jpeg,https://inaturalist-open-data.s3.amazonaws.com/photos/305781070/original.jpeg,https://www.inaturalist.org/photos/305781070,2023-07-30T12:42:10-07:00,Thomas Mesaglio,iNaturalist,http://creativecommons.org/licenses/by/4.0/,Thomas Mesaglio,305781070
```

**Comment by adam-collins** [comment]
Now that all images for an occurrence are displayed in the BIE gallery tabs I expect the order to be preserved rather than just the last image displayed in the BIE gallery tab (if that was the case).

**Comment by adam-collins** [comment]
On further investigation it appears the order is not preserved when put into SOLR. The image order 1,2,3 becomes 3,1,2 in this example https://biocache.ala.org.au/occurrences/44e45da8-ba26-4c1f-9d6d-8a241114b98e

It could be achieved by creating a new SOLR field that contains the ordering information and updating every gallery and image display page. If continuing, examples of how iNaturalist provides user reordered image information to us is required as the above example only indicates what looks like a default image ordering, numerical ascending. I don't think it is worth the effort given that we no longer pick one image to display but rather include them all (per occurrence) in the BIE gallery. Be aware that reloading the BIE gallery reorders the gallery images.

In summary the requirements of the originating issue are already met, that is the display of the first iNaturalist image in the BIE gallery.

**Comment by TaniaGLaity** [comment]
duplicate of [bie-plugin #247](https://github.com/AtlasOfLivingAustralia/bie-plugin/issues/247)

**Comment by TaniaGLaity** [comment]
potentially too much work for too little gain at this stage. cannot preserve order from source currently - requires lots of work in the backend to make this change. would require an additional field
